src/domain/user/helper/UserHelper.ts

```diff
@@ -1,55 +1,49 @@
-import Iuser from '../types/UserTypes'
+import IUserHelper from '../../../interfaces/IUserHelper'
+import IUser from '../../../interfaces/IUser'
 
-class UserHelper {
- static checkIfEquals(dados: string, user: Iuser[]) {
-  const cpfUser = Object.values(user)
-  const lista = cpfUser.map((item) => item.email)
-  lista.forEach((lista) => {
-   if (lista === dados) {
-    throw new Error('Email já cadastrado')
-   }
-  })
-  return true
- }
+export default class UserHelper implements IUserHelper {
+  checkIfEquals (dados: string, database: IUser[]): void {
+    const emailUser = Object.values(database)
+    const lista = emailUser.map((item) => item.email)
+    if (lista.includes(dados)) {
+      throw new Error('Email já cadastrado')
```

Comment:
Qual e-mail já foi cadastrado?
```suggestion
      throw new Error(`E-mail: ${email} já cadastrado!`)
```

Reply:
Feito! Valeuu